weave

```diff
@@ -32,6 +32,7 @@ usage() {
     echo "weave status"
     echo "weave version"
     echo "weave stop"
+    echo "weave put-dns    <cidr> [<cidr> ...] <container_id> [-fqdn fqdn]"
```

Comment:
The `-h` option to `weave expose` mirrors the `-h` that can be passed to `weave run` to set the weaveDNS name at container launch, unfortunately that is interpreted by Docker itself and cannot be changed to follow suit...